### PR TITLE
Swipe issue fix

### PR DESCRIPTION
**Pull request by moorea**
-Sexy pink nav bar
-"TM" logo instead of our happy face muncher
-Swiping through carousel now works smooth as silk :sunglasses:

### CODE REVIEW

File: TruckMuncher/MapViewController.swift [hunk 1]
```diff
@@ -305,11 +301,13 @@ class MapViewController: UIViewController,
...
-        
-        if (annotations.count > 0) {
-            let curIndex = truckCarousel.currentItemIndex
-            centerMapOverCoordinate(annotations[curIndex].coordinate)
+        for i in 0..<activeTrucks.count {
+            let currentCarouselTruckId = (activeTrucks[carousel.currentItemIndex] as RTruck).id
+            let someTruck = activeTrucks[i] as RTruck
+            if someTruck.id == currentCarouselTruckId {
+                let coord = CLLocationCoordinate2D(latitude: someTruck.latitude, longitude: someTruck.longitude)
+                centerMapOverCoordinate(coord)
```
Comment (jault3) [inline TruckMuncher/MapViewController.swift]: you could throw a break in here to avoid iterating the rest of the list

**Comment by jault3** [comment]
Need to merge master into this before you can go ahead with the merge, but it looks pretty good. Ill review quickly after you pull in master

**Comment by moorea** [comment]
Pulled in the changes from Master

File: TruckMuncher/MapViewController.swift [hunk 2]
```diff
@@ -162,12 +162,10 @@ class MapViewController: UIViewController,
         var clusterAnnotation = view.annotation as? CCHMapClusterAnnotation
         var truckLocationAnnotation = clusterAnnotation?.annotations.allObjects[0] as? TruckLocationAnnotation
         
-        
         if let tappedTruckIndex = truckLocationAnnotation?.index {
             truckCarousel.currentItemIndex = tappedTruckIndex
             truckCarousel.scrollToItemAtIndex(tappedTruckIndex, animated: true)
             
-            centerMapOverCoordinate(truckLocationAnnotation!.coordinate)
```
Comment (jault3) [inline TruckMuncher/MapViewController.swift]: we need this

Comment (moorea) [inline TruckMuncher/MapViewController.swift]: Sorry, i thought you were talking about the instance in carouselCurrentItemIndexDidChange() b/c that's what GitHub showed me when I clicked that link. 

No, this call is no longer needed.

**Comment by jault3** [comment]
:shipit:

**Pull request merged**